src/decaylanguage/decay/viewer.py

```diff
@@ -12,16 +12,13 @@
 
 import itertools
 
-try:
-    counter = itertools.count().__next__
-except AttributeError:
-    counter = itertools.count().next
-
 import graphviz
 
 from particle import latex_to_html_name
 from particle.converters.bimap import DirectionalMaps
 
+counter = iter(itertools.count())
+
```

Comment:
Use the `next()` builtin instead, that's calls the right thing on both Pythons. Also call `iter` if you plan to manually iterate something.